scissors/game_logic.c

```diff
@@ -0,0 +1,70 @@
+/**
+ * @file       game_logic.c
+ * @brief      game_logic
+ * 
+ * @author     Leonid Chebenko
+ * @version    1.0
+ */
+
+#include<stdio.h>
+#include<stdlib.h>
+#include<unistd.h>
+#include "game_logic.h"
+
+extern int computer;
+extern int player;
+extern void start();
+
+void game_logic()
+{
+        int game_continue;
+
+        computer = rand()% 3+1;
+
+        printf("Rock...");
```

Comment:
Due to the default line buffering, all these lines will be printed together after the first string with `'\n'` symbol is printed. So you have a weird 5 seconds delay before printing of all the texts.
Add `fflush(stdout)` to get desired behavior.